org.godotengine.Godot.json

```diff
@@ -147,8 +142,11 @@
             "build-commands": [
                 "install -D -m755 godot.sh /app/bin/godot",
                 "install -Dm644 org.godotengine.Godot.desktop /app/share/applications/org.godotengine.Godot.desktop",
-                "install -Dm644 org.godotengine.Godot-64.png /app/share/icons/hicolor/64x64/apps/org.godotengine.Godot.png",
-                "install -Dm644 org.godotengine.Godot.png /app/share/icons/hicolor/128x128/apps/org.godotengine.Godot.png",
+                "install -Dm644 org.godotengine.Godot.svg /app/share/icons/hicolor/scalable/apps/org.godotengine.Godot.svg",
+                "for size in {16,22,24,32,48,64,72,96,128,192,256,512}; do \\
```

Comment:
Most of these sizes are probably overkill. The small ones will go unused as its not like there is a tray icon. and the odd-large sizes like 72, 96, 192, are likely never used anywhere either.